### DISCUSSION TITLE
Can specific property be filtered out from the code example?

### DISCUSSION THREAD

**Discussion by ajkl2533** [storybookjs/storybook#29819] [Help]
### Summary

I have a Datatable component documented in Storybook. I have multiple stories build around it's functionality, but in all stories' code examples we have duplicated the `columns` (column shape definition) and `data` properties. Those two props are super lengthy (as you can see in the screenshot) and not very useful to overall documentation as users already know that you just need those. 

Is there a way how to completely remove specific properties from code examples or replace those with ellipsis like `data={...}`? 

<img width="636" alt="Screenshot 2024-12-05 at 10 45 57" src="https://github.com/user-attachments/assets/51eeb279-3e8d-49f3-aa15-190640c62036">

**Comment by smlimasu** [comment]
In the preview, meta or story level you can set the parameters.docs.source.transform method where you have the code as string and the storycontext object. You can manipulate the string like (pseudo code) if context.argtypes contains columns ... replace (regex) columns = {your data} with {...} or even trim it. Then return the adjusted code string. When the defaults are not enough, set language to typescript and format to HTML or typescript and code to dynamic (you can play around). 

If you need an example just ask.
